packages/builder-tools/connect-utils/types.ts

```diff
@@ -58,6 +58,7 @@ export type ConnectRuntimeEnv = {
   PH_CONNECT_VERSION_CHECK_INTERVAL: string;
   PH_CONNECT_CLI_VERSION: string;
   PH_CONNECT_FILE_UPLOAD_OPERATIONS_CHUNK_SIZE: string;
+  PH_CONNECT_DISABLE_LOCAL_PACKAGES: string;
```

Comment:
I think this doesn't need to be added to the runtime. I would name it `PH_DISABLE_LOCAL_PACKAGE`.